lib/cli.js

```diff
@@ -83,15 +84,27 @@ module.exports = class Cli {
    * bootstrap(config).then(lando => console.log(lando));
    */
   defaultConfig() {
+    const appRoot = this.getAppDir();
     const srcRoot = path.resolve(__dirname, '..');
     const version = require(path.join(__dirname, '..', 'package.json')).version;
+    const configSources = [
+      path.join(srcRoot, 'config.yml'),
+      path.join(this.userConfRoot, 'config.yml'),
+    ];
+    const pluginDirs = [srcRoot, this.userConfRoot];
+
+    if (appRoot) {
+      configSources.push(path.join(appRoot, '.lando.yml'));
```

Comment:
I don't think you want to merge app specific `.lando.yml` settings with global lando `config.yml` settings. This will cause unexpected behavior.

Reply:
The only key we really need is `plugins`. Maybe we can try something like: 
```
const appConfigSources = [];
if (appRoot) {
      appConfigSources.push(path.join(appRoot, '.lando.yml'));
}
```

And then in [bootstrap.js L51](https://github.com/lando/lando/blob/master/lib/bootstrap.js#L51)
```
  // Include app specific plugins as well
  if (!_.isEmpty(config.appConfigSources)) {
    const appConfig = helpers.loadFiles(config.appConfigSources);
    const appPlugins = appConfig.plugins;
    config = helpers.merge(config.plugins, appPlugins);
  }
```

Reply:
I have tested your suggestion locally and it worked out well. 

Just a note, the plugin suggested in my opening comment (`yarn add lando-plugin-magento-recipe`) needs an update pushed because it is no longer compatible with newer versions of Lando.